avae/train.py

```diff
@@ -44,6 +48,9 @@ def train(
     recon_fn,
     use_gpu,
     model,
+    gaussian_blur,
+    normalise,
+    shift_min,
```

Comment:
Add to shift-mean to docstring.

Reply:
fixed in [5876b00](https://github.com/alan-turing-institute/affinity-vae/pull/144/commits/5876b00ca5e0015fecbeed2d7376a3eb90f58af9)